### ISSUE TITLE
[Enhancement]: Rewatch once when session keep alive fail

### ISSUE THREAD

**Issue by wayblink** [milvus-io/milvus#23660]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What would you like to be added?

If etcd master node crash, etcd cluster can continue to work but all keep alive will disconnect. Hence, milvus nodes will quit if etcd master node crash. We need to try re-register once when keep alive fail.

**Comment by xiaofan-luan** [comment]
Good catch, If etcd failed for some reason, we have to either fail fast or rewatch .

**Comment by xiaofan-luan** [comment]
keep it

**Comment by xiaofan-luan** [comment]
@wayblink any progress?
